mylyn.tasks/connectors/gitlab/org.eclipse.mylyn.gitlab.ui/src/org/eclipse/mylyn/gitlab/ui/GitlabImages.java

```diff
@@ -0,0 +1,38 @@
+/*******************************************************************************
+ * Copyright (c) 2023 Frank Becker and others.
+ * 
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * https://www.eclipse.org/legal/epl-2.0
+ * 
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Frank Becker - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.mylyn.gitlab.ui;
+
+import org.eclipse.jface.resource.ImageDescriptor;
+import org.eclipse.jface.resource.ImageRegistry;
+import org.eclipse.swt.graphics.Image;
+
+public class GitlabImages {
+
+    // For the images
+    private static ImageRegistry fImageRegistry = new ImageRegistry();
+
+    public static String GITLAB_PICTURE_FILE = "icons/obj20/gitlab.png"; //$NON-NLS-1$
+
+    static {
+	fImageRegistry.put(GITLAB_PICTURE_FILE, GitlabUiActivator.getImageDescriptor(GITLAB_PICTURE_FILE));
```

Comment:
This is legal call, but the direction looks strange a bit. 
We either should make this class independent from `GitlabUiActivator` or we should use `GitlabUiActivator` to access image registry.

Reply:
Yes this class is deleted in the next commit.